lib/id3tag/frames/v2/frame_fabricator.rb

```diff
@@ -24,6 +24,8 @@ def frame_class
             UserTextFrame
           when /^T/
             TextFrame
+          when /^WXXX$/
```

Comment:
Looks like there can be [other URL frames](https://mutagen-specs.readthedocs.io/en/latest/id3/id3v2.4.0-frames.html#url-link-frames). like WCOM, WCOP, WPUB.. 

Would you like to add support for those as well?

Reply:
👍 Done in [e649ef0](https://github.com/krists/id3tag/pull/39/commits/e649ef01a04487d14d8e1f9d1a85b801a5f6ad6b)